python/lsst/meas/extensions/psfex/psfexPsfDeterminer.py

```diff
@@ -38,6 +38,44 @@
 from lsst.pipe.base import AlgorithmError
 
 
+class PsfexNoStarsError(AlgorithmError):
+    """Raised if no stars are available for PSF determination.
+
+    Parameters
+    ----------
+    msg : `str`
```

Comment:
Not a huge deal, but I would instead keep the message internal to this exception, since it's directly connected to the exception (instead of needing to be different for different places the exception is used).